src/lib/mapping/signMessage.ts

```diff
@@ -17,5 +20,6 @@ export async function signMessage(
 
   const signedMessage = await signature(hash);
 
+  // TODO: Why did we chose to return an array instead of the Buffer?
   return Array.from(new Uint8Array(signedMessage));
```

Comment:
The output of `signMessage` can be passed to `verify(...)`. Types should match. See https://docs.arconnect.io/api/sign-message.